autoload/neomake/makers/ft/crystal.vim

```diff
@@ -0,0 +1,16 @@
+function! neomake#makers#ft#crystal#EnabledMakers()
```

Comment:
https://github.com/embear/vim-localvimrc is awesome - and I like it for project specific settings.

https://github.com/editorconfig/editorconfig-vim#readme would be for `.editorconfig`, but I do not like the overhead.

We have really messy style in the maker vim files, and therefore it does not really matter (currently) what the style there is.
Adding a modeline, which takes a lot of relative space in general, does not help here.
We should rather enforce some linting here, if that becomes an issue.